src/Graph.spec.ts

```diff
@@ -1,15 +1,15 @@
 import expect from "expect";
 import { Graph, Board } from "./Types";
 import { getInitialBoard } from "./Board";
-import { boardToGraph } from "./Graph";
+import { boardToGraph, positionToCoordinate, moveMarbleInDirection } from "./Graph";
 
 const INITAL_BOARD: Board = [
   [1, 1, 1],
   [2, 0, 1],
   [1, 1, 1],
 ];
 
-describe("boardToGraph", () => {
+describe("Graph test", () => {
   it("should return a well formed Graph when a Board is passed as argument with the boardToGraph method", () => {
     const GRAPH: Graph = boardToGraph(INITAL_BOARD);
```

Comment:
name in lowercase

Reply:
We will wait until we have no overlapping development to rename our files